Jenkinsfiles/openedx-release-ci

```diff
@@ -0,0 +1,45 @@
+pipeline {
+    agent { label "jenkins-worker" }
+    options {
+        timestamps()
+        timeout(60)
+    }
+    environment {
+        GITHUB_CREDS = credentials('EDX_STATUS_BOT_CREDENTIALS')
+    }
+    stages {
+        stage('Git checkout') {
```

Comment:
I am confused here. Why are you cloning the master branch of edx/repo-tools. When you run the job, the small DSL stub does this clone already (see: https://github.com/edx/jenkins-job-dsl/pull/483/files#diff-3d3567ebbb08779f0c357f3d66247319R32). Is this redundant, or is there something specific to this process that I am missing?

Reply:
Nice catch..